### ISSUE TITLE
plot.netsim default color specification

### ISSUE THREAD

**Issue by smjenness** [statnet/EpiModel#325]
@connor-vanmeter would you please work on this in the next couple weeks?

There is a problem that the default colors for the mean lines and quantile bands do not show up when we explicitly specify the plot output with `y` when that output length is greater than 3 variables. The plot palettes should draw the the longer vector of palette colors in cases where the user wants to plot 4 or more variables in a single plot.

Case below:

```r
nw <- network.initialize(n = 100, directed = FALSE)
formation <- ~edges
target.stats <- 50
coef.diss <- dissolution_coefs(dissolution = ~offset(edges), duration = 20)
est1 <- netest(nw, formation, target.stats, coef.diss, verbose = FALSE)

# Epidemic model
param <- param.net(inf.prob = 0.3)
init <- init.net(i.num = 10)
control <- control.net(type = "SI", nsteps = 100, nsims = 5, verbose.int = 0)
mod1 <- netsim(est1, param, init, control)

# Four variables are able to be plotted, but the final si.flow does not display
mod1
plot(mod1, y = c("s.num", "i.num", "num", "si.flow"), legend = TRUE)

# Here we explicitly specify the plot colors, and all four variables display
plot(mod1, y = c("s.num", "i.num", "num", "si.flow"), mean.col = 1:4, qnts.col = 1:4, legend = TRUE)
```

**Comment by connor-vanmeter** [comment]
Sure Sam, will get started on it

**Comment by smjenness** [comment]
Closed with #327
